shell/platform/android/platform_view_android.cc

```diff
@@ -10,14 +10,10 @@
 #include "flutter/fml/synchronization/waitable_event.h"
 #include "flutter/shell/common/shell_io_manager.h"
 #include "flutter/shell/gpu/gpu_surface_gl_delegate.h"
-#if IMPELLER_SUPPORTS_PLATFORM
```

Comment:
This was done as a hack to make sure that google3 would build.

Can we make sure build_engine=1 still works with this patch applied in google3?

Reply:
I expect the rules already ported for the Metal backend for ImpellerC to also work with the OpenGL backend (and others).

Reply:
The g3 build hardcodes `IMPELLER_SUPPORTS_PLATFORM` when depending on impeller. An internal follow-up CL would remove the define from the impeller target.